packages/cli/src/__tests__/fixtures/config.ts

```diff
@@ -25,6 +25,7 @@ export const configFixture: Config = {
     async3: {},
     arazzo1: {},
     overlay1: {},
+    catalogEntity1: {},
```

Comment:
I don't think it's appropriate to add catalog entities format to the list of API specifications. I believe it should reside separately, more like we treat `redocly.yaml` format.